pkg/security/utils/cgroup.go

```diff
@@ -49,6 +50,43 @@ func (cg ControlGroup) GetContainerID() containerutils.ContainerID {
 	return containerutils.ContainerID(id)
 }
 
+// GetLastProcControlGroups returns the first cgroup membership of the specified task.
+func GetLastProcControlGroups(tgid, pid uint32) (ControlGroup, error) {
+	data, err := os.ReadFile(CgroupTaskPath(tgid, pid))
+	if err != nil {
+		return ControlGroup{}, err
+	}
+
+	data = bytes.TrimSpace(data)
+
+	index := bytes.LastIndexByte(data, '\n')
+	if index < 0 {
+		index = 0
+	}
+	firstLine := string(data[index:])
```

Comment:
Should we name this var "lastLine" instead ?

Reply:
ah yeah sorry..